hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/container/ContainerCommands.java

```diff
@@ -93,6 +93,8 @@ public class ContainerCommands implements Callable<Void>, SubcommandWithParent {
   @Spec
   private CommandSpec spec;
 
+  private String dataPath;
```

Comment:
Can we just define `@CommandLine.Option` here and use instead of duplicating code in every subcommands?

Reply:
+1

Reply:
> This led to command structure issues earlier
This is anyway a new option so we can place it anywhere without bothering about any compatibility issues. Why is it a problem?